manageiq-gems-pending.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = "> 2.4"
 
-  s.add_runtime_dependency "activesupport",           "~> 6.0"
+  s.add_runtime_dependency "activesupport",           ">=6.0"
```

Comment:
I'm debating just keeping this more open ended, allowing us to just test the versions we care about but also making it easier to start on 7.1 without having to update this gem immediately.  The application controls the upper bound for the version constraint so I don't think we need to do `~>` here.

Reply:
Yeah that works for me, we could always add an upper bound if we discover a hard incompatibility with a newer rails version (like we found with manageiq-loggers and Broadcast)